cumulusci/tasks/salesforce/UpdateDependencies.py

```diff
@@ -78,6 +83,15 @@ def _init_options(self, kwargs):
                 f"Unsupported value for security_type: {self.options['security_type']}"
             )
 
+        if "ignore_dependencies" in self.options:
+            if any(
+                "github" not in dep and "namespace" not in dep
+                for dep in self.options["ignore_dependencies"]
+            ):
+                raise TaskOptionsError(
+                    "An invalid dependency was specified for ignore_dependencies."
```

Comment:
This isn't a blocker, but this error would be more helpful if we show the input that was invalid and explain why it was invalid.